dartagnan/src/test/resources/VULKAN-DR-NOCHAINS-expected.csv

```diff
@@ -0,0 +1,6 @@
+litmus/VULKAN/NOCHAINS/mp3transitive.litmus,1
+litmus/VULKAN/NOCHAINS/mp3transitive2.litmus,1
+litmus/VULKAN/NOCHAINS/mp3transitive3.litmus,1
+litmus/VULKAN/NOCHAINS/mp3transitive4.litmus,1
+litmus/VULKAN/NOCHAINS/mp3transitivefail.litmus,1
+litmus/VULKAN/NOCHAINS/mp3transitivefail2.litmus,1
```

Comment:
I just noticed that [some tests](https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/master/alloy/tests/mp3transitive.test) are to be run both with and without chains, thus we should keep them all in the base `VULKAN` folder (sorry about this). Also, be sure to add those tests to both `VULKAN-expected` and `VULKAN-DR-NOCHAINS` (the name of the later should not have the `DR` I think)

Reply:
Moved, but because all of them are expected to have data racy, I think only the `DR-NOCHAINS` expected result is needed?

Reply:
I think we can actually also check them wrt `program_spec` so we need to add a new test class and the corresponding expected file